test/_globals-test.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const supertest = require('supertest');
+const fs = require("fs");
+
+const config = require('wild-config');
+
+const server = supertest.agent(`http://127.0.0.1:${config.api.port}`);
+
+const titles = [];
+const unsupportedTitles = [];
+
+// global beforeEach to run before EVERY test
+beforeEach("Get test data before each test", async function () {
+    const currentTestTitle = this.test.ctx.currentTest.title; // eslint-disable-line no-invalid-this
+    if (/POST|PUT|DELETE|GET|post|put|delete|get/.test(currentTestTitle) && /success|failure/.test(currentTestTitle)) {
+        titles.push(currentTestTitle);
+    } else {
+        unsupportedTitles.push(currentTestTitle);
+    }
+});
+
+// eslint-disable-next-line no-undef
+after("Generate test overview table after all tests", async () => {
+    try {
```

Comment:
Maybe not use `try..catch` here. If there is an exception, the `after()` handler will catch it and log it as a failure.

Reply:
removed try catch